udrzba/nasad.php

```diff
@@ -21,7 +22,7 @@
 exec('git rev-parse --abbrev-ref HEAD', $out);
 $vetev = $out[0];
 if (!($vetev === 'master' || strpos($vetev, 'redesign') === 0)) {
-  echo "notice: you're not on automatically deployed branch, deployment skipped\n";
+  echo "You're not on automatically deployed branch, deployment skipped\n";
```

Comment:
Nějak mi přišlo divné, že mi to tvrdí, že se stala _notice_ a přitom mi selhal celý deploy.